Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -68,6 +69,14 @@ foreach ($suite in $suites)
     }    
 }
 
+$testMachinesGroupName=$testMachineGroup
+if ([string]::Equals($env:UseDtaV2Agent, "true", [System.StringComparison]::OrdinalIgnoreCase)) {
```

Comment:
Yes it does

---
In reply to: [104138653](https://github.com/Microsoft/vsts-tasks/pull/3727#discussion_r104138653) [](ancestors = 104138653)